--- proj1.c	2019-12-01 20:09:51.336728000 +0100
+++ proj1_new.c	2019-12-01 20:11:08.970594000 +0100
@@ -103,7 +103,7 @@
 			case 'Y': pismeno = '9'; break;
 			case 'Z': pismeno = '9'; break;
 			case '+': pismeno = '0'; break;
-			default: pismeno = pismeno; break;
+			default: break;
 		}
 		str1[i] = pismeno;                                    // do pomocneho stringu ukladam znaky ktore uz presli switchom (zmenili sa na ciselne znaky)
 	}
@@ -229,7 +229,7 @@
 			}
 			if (found == false)									    // found = false, pretoze nenaslo ani jednu zhodu
 														    // v celom telefonnom zozname
-				fprintf(stdout, "Not found!\n");						    // vypise ze nic nenaslo
+				fprintf(stdout, "Not found\n");						    // vypise ze nic nenaslo
 		}
 			
 	}		
